File: 백준/Bronze/11022. A＋B － 8/A＋B － 8.py
number = int(input());
caseList = [];

for i in range(number) :
    a = input().split(' ');
    caseList.append(a);

for i in range(number) :
    print('Case #' + str(i +1) +': '+ caseList[i][0] + ' + '+ caseList[i][1] +' = ' + str(int(caseList[i][0]) + int(caseList[i][1])));
    

File: 백준/Bronze/11022. A＋B － 8/A＋B － 8.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringBuilder result = new StringBuilder();
        int n = Integer.parseInt(br.readLine());
        for (int i = 1; i <= n; i++) {
            String[] input = br.readLine().split(" ");
            int a = Integer.parseInt(input[0]);
            int b = Integer.parseInt(input[1]);
            result.append("Case #")
                .append(i).append(": ")
                .append(a).append(" + ")
                .append(b).append(" = ")
                .append(a + b)
                .append('\n');
        }
        System.out.println(result);
    }
}
